packages/ai-jsx/src/react/completion.tsx

```diff
@@ -1,19 +1,19 @@
-/** @jsx AI.createElement */
-import * as AI from './index.js';
+import * as LLMx from './core.js';
 import React from 'react';
 import { SystemMessage, UserMessage } from '../core/completion.js';
 import { JsonChatCompletion } from '../batteries/constrained-output.js';
+import { isJsxBoundary } from './jsx-boundary.js';
 
 function reactComponentName(component: React.JSXElementConstructor<any> | string) {
   return typeof component === 'string' ? component : component.name;
 }
 
 export async function UICompletion(
-  { example, children }: { example: React.ReactNode; children: AI.Node },
-  { render, logger }: AI.ComponentContext
+  { example, children }: { example: React.ReactNode; children: LLMx.Node },
+  { render, logger }: LLMx.ComponentContext
 ) {
   const reactComponents = new Set<React.JSXElementConstructor<any> | string>();
-  function collectComponents(node: React.ReactNode | AI.Node, inReact: boolean) {
+  function collectComponents(node: React.ReactNode | LLMx.Node, inReact: boolean) {
```

Comment:
I thought this was the opposite direction of where we wanted to be moving?

Reply:
It is, but:
- Typescript preserves comments by default (and I don't think we want to change that)
- Babel complains about the JSX pragmas

So this is a way to kick the can until the entire `ai-jsx` codebase is aligned on using `AI.createElement` (so as not to require internal pragmas), or even better, moved over to the new `_jsx()`-style conversion (in which case the leftover pragmas will be meaningless because the JSX is already converted by `tsc`).

Reply:
Ok, if this is a can kick, that's fine with me.